packages/discord.js/src/util/Events.js

```diff
@@ -108,7 +108,7 @@ module.exports = {
   ChannelDelete: 'channelDelete',
   ChannelPinsUpdate: 'channelPinsUpdate',
   ChannelUpdate: 'channelUpdate',
-  ClientReady: 'ready',
+  ClientReady: 'clientReady',
```

Comment:
This feels to me like a breaking change.

Reply:
We didn't do this for Events.WebhooksUpdate which still has the value webhookUpdate in v14 despite it being marked as deprecated the same way.

Reply:
Its not really a breaking change unless you, for whatever reason, did `Events.ClientReady === "ready"`, which at that point I'd question what you're doing. In fact I'd even argue its a correction, all other enum names can easily be converted to enum keys, not this one